verticaltabs.js

```diff
@@ -168,12 +144,27 @@ VerticalTabs.prototype = {
       OldPrintPreviewListenerExit();
     };
 
-    //reset _lastRelatedTab on changing preferences
+    let tabs = document.getElementById('tabbrowser-tabs');
+    let openTabsTop = !prefs.opentabstop;
```

Comment:
this variable seems to never be actually used? Every time it is checked it seems `prefs.opentabstop` is also checked...